mesonbuild/mtest.py

```diff
@@ -903,8 +903,7 @@ def num(self) -> int:
             self._num = TestRun.TEST_NUM
         return self._num
 
-    @property
-    def detail(self) -> str:
```

Comment:
Please explain in the commit message why this is an improvement or why it is required for the rest of the changes.

Reply:
Hopefully this is better now. I tried to improve the commit messages all around...